### DISCUSSION TITLE
Lots of Errors today from STT and TTS

### DISCUSSION THREAD

**Discussion by patrickml** [deepgram/community#792] [General help] [answered]
### Which Deepgram product feature are you using?

Deepgram API - STT Streaming

### Details

today we received a lot of errors from both deepgram services. I checked our logs two weeks back and they all happened today throughout the day. We have not made any environmental changes. Please see the photos below 

![IMG_8354](https://github.com/deepgram/community/assets/7581369/958ec04c-eb26-4fde-bb43-01c8a93a3583)
![IMG_8353](https://github.com/deepgram/community/assets/7581369/a349e96b-2d27-4475-b119-415f1a698e47)

![IMG_8352](https://github.com/deepgram/community/assets/7581369/7f0a9fff-48e4-4f68-aa67-6cc4e2d62a6c)
![IMG_8351](https://github.com/deepgram/community/assets/7581369/94435f14-685f-4abc-96af-87e150af5a09)

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment, marked as answer]
@patrickml, apologies for these issues you've been experiencing with Deepgram. 

There are several backbone internet issues that we are aware of. To help root-cause your issue, could you please share the following:
- What country or region are you making requests from? (There is a known network issue with India/southeast Asia)
- If you are sending URLs to audio files (rather than raw audio) to Deepgram, where is your audio stored? (There is a known but mitigated network issue with Google Cloud Storage)
- Can you please run this command and share the results: `traceroute api-alt.md1.deepgram.com`
